src/aosm/azext_aosm/tests/latest/tests_utils.py

```diff
@@ -0,0 +1,42 @@
+# # --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# # --------------------------------------------------------------------------------------------
+
+import os
+from typing import Dict
+
+from jinja2 import Template
+
+
+def get_tests_path():
+    code_directory = os.path.dirname(__file__)
+    tests_directory = os.path.join(code_directory, "../")
+    return tests_directory
+
+
+def update_input_file(input_template_name, output_file_name, params: Dict[str, str]):
```

Comment:
I see why you've written these, but as in my other comment, if we need them that suggests we're not really unit testing any more.

I think leave them for now, but use them sparingly, and consider mocking out anything that isn't the object under test.